mne/viz/evoked.py

```diff
@@ -311,10 +311,18 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                         for i in bad_ch_idx:
                             if i in idx:
                                 colors[idx.index(i)] = 'r'
-                    for ch_idx in range(len(D)):
+
+                    # find the channels with the least activity,
+                    # and map them in front of the more active ones
+                    auc = np.trapz((np.abs(D - D.mean(1).reshape(-1, 1))))
```

Comment:
Let's first decide on this actually being an improvement at all before deciding on the implementation, I would think ...
It works well with the standard deviation, so much I can say at least.

I think we possibly should make SOME principled decision about how to z-order these, though.